### DISCUSSION TITLE
How to load the provided packages?

### DISCUSSION THREAD

**Discussion by NMadamopoulos** [francois-rozet/sleek-template#2] [Q&A] [answered]
Hi Francois,

I found your very interesting and "sleek" template.  I down loaded the zip file in overleaf and tried to experiment with it.  In particular, I liked the framed examples you show in your pdf.  However, when I try to set a framed example it does not run.  It gives that the fmd-theorem is undefined.  I looked at the sleek-theorem and there is the following:
\declaretheorem[mdthm={style=thicc, linecolor=red!20, frametitlebackgroundcolor=red!20}, sibling=theorem, name=\lgtheorem]{fmd-theorem}

Do I have to run some set up first?

Thanks,
NM

**Comment by francois-rozet** [comment, marked as answer]
Hello @NMadamopoulos, thank you for your interest!

The `main.tex` in the archive you downloaded is quite minimal. In particular, it does not import the `sleek-theorems` and `sleek-listings` packages by default, but they are available. You can import them with

```latex
\usepackage{packages/sleek-theorems}
\usepackage{packages/sleek-listings}
```

**Reply by NMadamopoulos** [reply]
Great, thanks.  I tried it and worked.  I have to experiment more.  If I want to use only the framed theorems etc in another project, do I only copy sleek-theorem.sty and sleek.sty?  what should I include in the commands and/or usepackage?  Best-NM

**Reply by francois-rozet** [reply]
The four packages are independent. So you can copy and include (`usepackage`) any of them arbitrarily.

**Reply by NMadamopoulos** [reply]
Thanks!

**Reply by NMadamopoulos** [reply]
By the way, I am trying to understand the structure (not very familiar with the coding) of the theorems.sty.  You have copied the same format to some of them, right?  Also, how do I remove the numbering from Theorem or Note etc?  Can I also have it include the chapter number, e.g., Note 1.2, for chapter 1 and Note 2.1 for chapter 2?  Thanks
